upstairs/src/upstairs.rs

```diff
@@ -1988,6 +2032,76 @@ impl Upstairs {
         }
     }
 
+    /// Start min-quorum reconciliation
+    fn on_min_quorum(&mut self) {
+        // Reconciliation only happens during initialization.
+        let UpstairsState::GoActive {
+            min_quorum_deadline,
+            ..
+        } = &mut self.state
+        else {
+            warn!(
+                self.log,
+                "min-quorum negotiation found upstairs state {:?}; cancelling",
+                self.state
+            );
+            return;
+        };
+        assert!(min_quorum_deadline.is_some());
+        *min_quorum_deadline = None;
```

Comment:
I think clearing `min_quorum_deadline` here means that if `ready_count` != 2 (checked below), control will leave this function and not attempt min quorum activation again?

Reply:
I agree, but I think this is fine: next time we end up with 2 ready Downstairs, then we'll reschedule `min_quorum_deadline`.